game_event_player/admin.py

```diff
@@ -0,0 +1 @@
+# from django.contrib import admin
```

Comment:
@ZvikaNaorCohen That's the place you meant to address your last comment I assume. 

anyway as naor said, GameEventPlayer entity Should be registered to admin here.